stanford-earth/matson
# Earth 1021 more social

# READY

# Summary
- Adding social media from Get Social to events and news. 
- NOT Spotlights yet since that php error isn't resolved.
- Wrap location text in a link using the MapURL

# Needed By (Date)
- Next push, or could be at same time as Spotlights (EARTH-1080)

# Urgency
- Not very, but would be nice.

# Steps to Test

1. Go to Manage Display for Events and drag the MapURL field into the new MapURL section.  Format it as a Link, make sure the Trim Length field is blank and you have checked "URL Only" and "Show URL as Plain Text."
2. Go to an event node. See that the teal social media icons have been replaced by the getsocial counts.
2.5. Check that the there is a URL in the MapURL field. If there is, check that the "Where" link goes there.
3. Go to a news node. See that the teal social media icons have been replaced by the getsocial counts.
4. Test both at various screensizes

# Affected Projects or Products
- News, Events

# Associated Issues and/or People
- EARTH-1021
- EARTH-1080
- EARTH-1064

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
